pkg/kf/testutil/integration.go

```diff
@@ -405,6 +406,82 @@ func KF(t *testing.T, kf KfInvoker) *Kf {
 	}
 }
 
+// CreateQuota creates a resourcequota.
+func (k *Kf) CreateQuota(ctx context.Context, quotaName string, extraArgs ...string) []string {
+	k.t.Helper()
+	Logf(k.t, "creating quota %q...", quotaName)
+	defer Logf(k.t, "done creating quota %q.", quotaName)
+
+	args := []string{
+		"create-quota",
+		quotaName,
+	}
+
+	output, _ := k.kf(ctx, k.t, KfTestConfig{
```

Comment:
instead of ignoring the errors, you should probably add them to the output of the function and check them when called to make sure the command doesn't accidentally fail and/or it fails as expected

Reply:
Elsewhere we are using `PanicOnError`

Reply:
I chose not to use `PanicOnError` for the quota functions bc when it panics, the deferred delete quota never gets called and the test quotas don't get cleaned up. I was planning on asserting that the error is nil so that if the test fails, the end state is still clean